asgard/handler.cpp

```diff
@@ -152,31 +152,31 @@ pbnavitia::Response Handler::handle_matrix(const pbnavitia::Request& request) {
     mode_costing.update_costing_for_mode(mode, request.sn_routing_matrix().speed());
     const auto costing = mode_costing.get_costing_for_mode(mode);
 
-    LOG_INFO("Projecting " + std::to_string(navitia_sources.size() + navitia_targets.size()) + " locations...");
-    const auto range = boost::range::join(navitia_sources, navitia_targets);
-    const auto projected_locations = projector(begin(range), end(range), graph, mode, costing);
+    // We use the cache only when there are more than one element in the sources/targets, so the cache will keep only stop_points coord
+    bool use_cache = (navitia_sources.size() != 1);
```

Comment:
```suggestion
    bool use_cache = (navitia_sources.size() > 1);
```
To be consistent with the comment above ?